adal/tests/Test.ADAL.NET.Unit.net45/Helpers/PlatformParametersFactory.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using System;
+
+namespace Test.ADAL.NET.Unit
+{
+    public class PlatformParametersFactory
+    {
+        public static IPlatformParameters CreateDefault()
+        {
+#if DESKTOP
+            return new PlatformParameters(PromptBehavior.Auto);
+#elif NET_CORE
+            return new PlatformParameters();
```

Comment:
In the end this wasn't needed, as all tests that create PP invoke the interactive flow which is not available on netcore. But still, it's a good basis for tests in other platforms.